esp32/door-sensor/src/main.cpp

```diff
@@ -0,0 +1,89 @@
+#include <Arduino.h>
+#include <HTTPClient.h>
+#include <WiFi.h>
+
+#include "config.h"
+
+#define DOOR_SENSOR_PIN 15
+#define DEBOUNCE_DELAY 500

```

Comment:
I would say to increase debounce delay significantly, at least to 30 seconds or so

Reply:
https://github.com/GryphonRacingFSAE/Discord-Bot/pull/20/commits/eef9ffd10c4e1efb937883f5f0b3b33e5a08300d